cmd/kubemark/hollow-node.go

```diff
@@ -145,8 +151,9 @@ func main() {
 	defer logs.FlushLogs()
 
 	if err := command.Execute(); err != nil {
-		os.Exit(1)
+		return err
```

Comment:
You have to see how to create `command` and consider that command.Execute() might do os.Exit() or klog.Fatalf. If command is created to do os.Exit() or klog.Fatal, you have to change it to return error instead of exit.

like this point
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubemark/hollow-node.go#L186

Reply:
done. :)